Doc/pages/workflow.rst

```diff
@@ -4,58 +4,80 @@
+Once you have converted your trajectory to the MDANSE MDT format,
+you can use it as input for all analysis types.
+See also :ref:`trajectory-converters`.
 
 2a. Analysis parameters
 -----------------------
-Most analysis jobs offered by MDANSE allow a number of different parameters
-which can be adjusted. We describe some of the more common parameters found
+
+Most analysis jobs offered by MDANSE are controlled by a number
+of input parameters which can be adjusted.
+MDANSE GUI typically attempts to fill all the fields with
+usable starting values, and highlights the invalid ones
+that need to be corrected by the user.
+We describe some of the more common parameters found
 in MDANSE below.
 
 Frames
 ^^^^^^
-It is not necessary to use all the time frames of your MD simulation. You can decide
-to limit the range of simulation time, and reduce the number of the frames taken in that
+
+You can decide to limit the range of simulation frames used in the calculations,
+or to reduce the number of the frames taken in that
 range by increasing the step between them. Only the frames you selected will
 be passed to the analysis job. See also :ref:`param-frames`.
 
+A more detailed discussion of the (correlation) frames can be found here:
+:ref:`correlation-frames`.
+
 Atom selection
 ^^^^^^^^^^^^^^
-Just as it is not necessary to include all the time frames in the analysis, it is also
-possible to select only a subset of all the atoms present in the trajectory. Once you
-have defined a selection, you can decide to run an analysis job on the selected atoms, and
-ignore the rest. This is useful if you are trying to determine which atoms contribute
-to a specific feature in your results. See also :ref:`param-atom-selection`.
+
+Just as it is not necessary to include all the time frames in the analysis,
+it is also possible to select only a subset of all the atoms present in the
+trajectory. Once you have defined a selection, you can run an analysis job
+on the selected atoms, and ignore the rest. This is useful if you are trying
+to determine which atoms contribute to a specific feature in your results.
+More details are given in the section :ref:`atom-selection`.
 
 Resolution
 ^^^^^^^^^^
-The resolution is enabled only for the analysis types which calculate an energy spectrum.
-This is normally applied to calculations involving Fourier transform of a correlation function.
-The resolution is applied by multiplying the time-dependent function with a window function
-before applying the Fourier transform. The details are given in the section
+
+The resolution is enabled only for the analysis types which calculate an
+energy spectrum.This is normally applied to calculations involving Fourier
+transform of a correlation function. The resolution is applied by multiplying
+the time-dependent function with a window function before applying the Fourier
+transform. The details are given in the section
 :ref:`param-instrument-resolution`.
```

Comment:
There is a section on the Fourier transforms and resolution functions here http://localhost:63342/MDANSE/Doc/_build/html/pages/correlation.html#fourier-spectrum. It might be worth referencing it here.